modules/rostests/kmtests/ntos_ex/ExFastMutex.c

```diff
@@ -310,6 +310,14 @@ TestFastMutexConcurrent(
 
 START_TEST(ExFastMutex)
 {
+#if defined(_M_AMD64)
+    if (TRUE)
+    {
+        skip(FALSE, "ROSTESTS-367: Skipping kmtest:ExFastMutex because it hangs on Windows Server 2003 x64-Testbot. Set winetest_interactive to run it anyway.\n");
```

Comment:
Misleading comment, `winetest_interactive` won't change anything for this kmtest